src/js/components/CharityPage.jsx

```diff
@@ -136,27 +128,78 @@ const CharityAnalysisSection = ({charity}) => (
+	
+		if (obj.inputs.findIndex(elem => elem.name === "projectCosts") !== -1) {
+			if (i >= colours.length) i = 0; //limiting colours to only those in the array
+			let insert = {
+				color: colours[i],
+				title: obj.name,
+				value: obj.inputs[obj.inputs.findIndex(elem => elem.name === "projectCosts")].value,
+			};
+			if (insert.value) outputarray.push(insert);
+			else i--;
+			i++;
+		}
+	});
+	const returnObj = {};
+	returnObj.totalProjectValue = outputarray.reduce((acc, elem) => acc + Number(elem.value), 0);
+	returnObj.year = mostRecentYear;
+	return returnObj;
+
+}
+
 const CharityAbout = ({charity}) => {
 	// Safety: in case the url is e.g. wwww.mysite.com with no http(s)
 	let churl = charity.url;
 	if (churl && churl.indexOf('http') !== 0) churl = 'http://'+churl;
+	let pieChartData = [];
+	let totalProjectValue = 0;
+	let mostRecentProjectYear;
+	if (charity.projects !== undefined) {
+		const projectData = getProjectData(charity, pieChartData);
+		totalProjectValue = projectData.totalProjectValue;
+		if (pieChartData.length > 0) {
+			mostRecentProjectYear = ` (${projectData.year})`;
+		}
+	}
+
 	return (
 		<div className='charity-about'>
-			{NGO.getName(charity) !== NGO.displayName(charity)? <h4 className='official-name'>{NGO.getName(charity)}</h4> : null}
+			{/* {NGO.getName(charity) !== NGO.displayName(charity)? <h4 className='official-name'>{NGO.getName(charity)}</h4> : null} */}
 			<CharityAboutImage charity={charity} />
-			<div className='descriptions'>
-				<div className='description-short'>
-					{charity.summaryDescription? <MDText source={charity.summaryDescription} /> : null}
-				</div>
-				<div className='description-long'>
-					{charity.description? <MDText source={charity.description} /> : null}
-				</div>
-			</div>
-			<div className='url'>
-				<a href={churl} target='_blank'>{charity.url}</a>
-			</div>
-			<div className='official-details'>
-				{NGO.registrationNumbers(charity).map(reg => <small key={reg.id}>{reg.regulator}: {reg.id}</small>)}
+			<div className='charity-about-details div-section-text border'>
+				<h3 className='header-section-title'><b>Details on {charity.name}</b></h3>
+				<p><b>Website:</b> <a href={churl} target='_blank'>{charity.url}</a></p>
+				{NGO.registrationNumbers(charity).map(reg => <p key={reg.id}><b>{reg.regulator}</b>: {reg.id}</p>)}
+				<p><b>Program Split{mostRecentProjectYear}:</b></p>
+				<ul>
+					{pieChartData.map(prog => <li key={prog.title} style= {{'color': prog.color}}>{prog.title} - {Math.round(Number(prog.value) * 100/totalProjectValue)}%</li>)}
+				</ul>
+				{pieChartData.length !== 0 ? <PieChart
+					data={pieChartData}
+					label={({ dataEntry }) => Math.round(dataEntry.percentage) + '%'}
+					labelStyle={(index) => ({
+						fill: pieChartData[index].color,
+						fontSize: '4px',
+						fontFamily: 'Tajawal',
+					})}
+					radius={40}
+					labelPosition={105}
+					/>:<p>Unknown</p>}
```

Comment:
No point showing "unknown" -- it doesn't add anything, and for single project charities it's slightly inaccurate. If we haven't got a project split, just omit the Program Split section.